cmake/android/android_gradle_projects.cmake

```diff
@@ -82,6 +82,13 @@ foreach(fname ${GRADLE_WRAPPER_FILES})
   install(FILES "${OpenCV_SOURCE_DIR}/platforms/android/gradle-wrapper/${fname}" DESTINATION "${ANDROID_INSTALL_SAMPLES_DIR}/${__dir}" COMPONENT samples ${__permissions})
 endforeach()
 
+# set build.gradle namespace 
```

Comment:
Please fix whitespace issues:

https://pullrequest.opencv.org/buildbot/builders/precommit_docs/builds/102951/steps/whitespace%20opencv/logs/stdio

(remove extra spaces in EOL)

Reply:
I didn't see it, it is done.